Iroha `2.0.0-pre-rc.20` update (#172)
hyperledger/iroha-javascript: .changeset/little-berries-explode.md

---
'@iroha2/data-model': major
---

**refactor!**: join all data-model types as `datamodel` namespace export

##### What is the change

> If previously all data-model types were imported directly from the package:

Reviewer: ```suggestion
Previously, all data-model types were imported directly from the package:
```

```ts
import { AccountId, DomainId } from '@iroha2/data-model'

const acc = AccountId({
  name: 'alice',
  domain_id: DomainId({
    name: 'wonderland',
  }),
})
```

Now all data-model types are joined into a single `datamodel` namespace:

```ts
import { datamodel } from '@iroha2/data-model'

const acc = datamodel.AccountId({
  name: 'alice',
  domain_id: datamodel.DomainId({
    name: 'wonderland',
  }),
})
```

##### Why the change was made

> For a couple of reasons:

Reviewer: I don't remember how it's done in other sections like this but I feel like you can remove this part (the header is enough)
```suggestion
```

> - There were too many root-level package exports, not always related to each other. Now it should be easier to distinguish data-model related exports from other utilities.

Reviewer: you can make this list item similar to the one below
```suggestion
- To make it easier to distinguish data-model related exports from other utilities as there used to be too many root-level package exports, not always related to each other.
```

> - To be more compatible with how data model is presented in Java SDK

Reviewer: ```suggestion
- To make data model more compatible with how it is presented in Java SDK
```
